type_templates/vrm.js

```diff
@@ -144,8 +259,27 @@ export default e => {
       localPlayer.setAvatarApp(app);
     };
 
+    app.updateQuality = async () => {
+      // const quality = getGfxSettingJSON('character').details;
+      // return await _setQuality(quality, app)
+      const quality = JSON.parse(localStorage.getItem('GfxSettings'));
+      return await _setQuality(quality?.character.details, app)
+    }
+
+    await app.updateQuality();
+
+    //prep any outstanding meshes
+    //may not need this yet

```

Comment:
Then delete it.

Reply:
it will be immediately needed in the PR's for crunched and sprite meshes.  I can delete it now, but it'll immediately be back in the next pr